portfolio/src/main/webapp/art-gallery.js

```diff
@@ -24,33 +24,36 @@ let getLatestOpenedImgIndex;
 let frames = document.querySelectorAll(".art-container");
```

Comment:
This doesn't seem to be used outside of the init function call, could you move this line into the `init()` method?